cmd/tether/utils_linux.go

```diff
@@ -90,6 +91,23 @@ func (t *osopsLinux) SetHostname(hostname string) error {
 	return nil
 }
 
+func pciToMAC(pciAddr string) (string, error) {
+	addrPath := "/sys/bus/pci/devices/" + pciAddr + "/net/*/address"
```

Comment:
The `*` makes me nervous.  Is it possible more than device will be listed?   Can you guard against it?